Makefile

```diff
@@ -27,9 +27,7 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 # IMAGE_TAG_BASE defines the docker.io namespace and part of the image name for remote images.
 # This variable is used to construct full image tags for bundle and catalog images.
 #
-# For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
-# parodos.dev/operator-bundle:$VERSION and parodos.dev/operator-catalog:$VERSION.
-IMAGE_TAG_BASE ?= quay.io/parodos-dev/orchestrator-controller
+IMAGE_TAG_BASE ?= quay.io/orchestrator/controller-manager
```

Comment:
Should we use `orchestrator-operator` as image name instead? if this is the default name generated by the SDK, and has a wider effect on the rest of the generated code, we can use it.

Reply:
It's defined here:
[[projectName: orchestrator-operator](https://github.com/jordigilh/orchestrator-helm-operator/blob/d7ee0f512ce88cfe76e79b216a184d6af6a86357/PROJECT#L11)](https://github.com/jordigilh/orchestrator-helm-operator/blob/d7ee0f512ce88cfe76e79b216a184d6af6a86357/PROJECT#L10-L12)

We can set the value we want anyway. I'll change it to `orchestrator-operator` as suggested.